src/box/lua/config/applier/box_cfg.lua

```diff
@@ -496,14 +496,35 @@ local function apply(config)
         --
         -- The RO/RW mode remains unchanged on reload.
         local am_i_bootstrap_leader = false
+        local recently_bootstrapped = false
+
         if is_startup then
             local instance_name = configdata:names().instance_name
+
+            recently_bootstrapped =
+                snapshot.get_path(configdata._iconfig_def) == nil 
+
             am_i_bootstrap_leader =
-                snapshot.get_path(configdata._iconfig_def) == nil and
+                recently_bootstrapped and
                 instance_name == configdata:bootstrap_leader_name()
+
             box_cfg.read_only = not am_i_bootstrap_leader
         end
 
+        if recently_bootstrapped then
+            local renew_interval =
+                instance_config:get(configdata._iconfig_def,
+                                    'failover.renew_interval')
+
+            box.internal.recently_bootstrapped = recently_bootstrapped
+
+            fiber.new(function()
+                fiber.sleep(renew_interval)
+                box.cfg({read_only = true})
+                box.internal.recently_bootstrapped = nil
+            end)
```

Comment:
I suggest to set a fiber name in the first statement of the fiber function.

Reply:
Applied.